### DISCUSSION TITLE
Is anyone running the latest Openhantek6022 on Win10-64 ?

### DISCUSSION THREAD

**Discussion by Rodoz** [OpenHantek/OpenHantek6022#224] [Q&A] [answered]
I have the Hantek5022 complete with the AC input mod but hard as I try I have not been able to successfully install or run the latest Openhantek in Window10-64 or Virtual Linux - Mint.   Persist as I may, days of trial and failure have me tearing my hair out.     I am at the point of ripping the mods out and going back to the windows software Hantek supplied.   Can anyone point me to some links that may assist please.      Regards Rodoz

**Comment by Ho-Ro** [comment, marked as answer]
> ... days of trial and failure ...

And after days without success, you end up saving a few minutes by neither giving a detailed description of your actions nor the exact error message - but that is exactly what would give us the information we need to help you.

> Can anyone point me to some links that may assist please.
 
- [Run OpenHantek6022](https://github.com/OpenHantek/OpenHantek6022#run-openhantek6022)
- [Windows USB access](https://github.com/OpenHantek/OpenHantek6022#windows-usb-access)
- [Microsoft Windows USB driver install (with Zadig)](https://github.com/OpenHantek/OpenHantek6022/blob/main/docs/build.md#microsoft-windows-usb-driver-install-with-zadig)
- [Signed WinUSB driver for Hantek 6022BE/BL](https://github.com/OpenHantek/OpenHantek6022/blob/main/docs/build.md#signed-winusb-driver-for-hantek-6022bebl)

**Comment by Rodoz** [comment]
Thank you for the links.
